tiledb/cloud/notebook.py

```diff
@@ -71,7 +71,8 @@ def download_notebook_to_file(
         tiledb_uri,
         storage_credential_name,
     )
-    with open(ipynb_file_name, "w") as handle:
+    vfs = tiledb.VFS()
```

Comment:
Probably need to pass the ctx into the VFS constructor, looks like some previous open calls may have been missing it.

Reply:
thanks @ihnorton !! can you tell me more about what you mean by "previous" -- ?

Reply:
Sorry, I missed that these calls are for local download - please disregard.